modules/snapaligner/paired/main.nf

```diff
@@ -0,0 +1,41 @@
+process SNAPALIGNER_PAIRED {
+    tag '$meta.id'
+    label 'process_high'
+
+    conda (params.enable_conda ? "bioconda::snap-aligner=2.0.1" : null)
+    container "${ workflow.containerEngine == 'singularity' && !task.ext.singularity_pull_docker_container ?
+        'https://depot.galaxyproject.org/singularity/snap-aligner:2.0.1--hd03093a_1':
+        'quay.io/biocontainers/snap-aligner:2.0.1--hd03093a_1' }"
+
+    input:
+    tuple val(meta), path(reads)
+    path index
+
+    output:
+    tuple val(meta), path("*.bam"), emit: bam
+    path "versions.yml"           , emit: versions
+
+    when:
+    task.ext.when == null || task.ext.when
+
+    script:
+    def args = task.ext.args ?: ''
+    def prefix = task.ext.prefix ?: "${meta.id}"
+
+    """
+    mkdir -p index
+    mv $index index/
```

Comment:
Why not just pass $index as an directory to the module in the first place?

Reply:
Because I didn't know the syntax 😇  and the output from `index` are just the files

Reply:
Suggestions to improve my knowledge and the code quality are welcome!

Reply:
Ok fair enough!

Reply:
No, BWA takes a diretory but uses a `find` command to find the relevant files to get the 'basename' and path for the reference genome.

Fine either way tbh